### PR TITLE
Add info about CleverTap repo ownership

### PR DESCRIPTION
**Pull request by joeynmq**
### Proposed changes
Our Mobile-CleverTap integrations are actually maintained by the CleverTap team:
- https://github.com/CleverTap/clevertap-segment-android
- https://github.com/CleverTap/clevertap-segment-ios

Add this information to our public documentation so that our customers can contact the appropriate team for assistance.

### Merge timing
ASAP once approved

### CODE REVIEW

File: src/connections/destinations/catalog/clevertap/index.md [hunk 1]
```diff
@@ -7,12 +7,18 @@ cmode-override: true
...
+CleverTap supports the `identify`, `track`, `page` (server-side only), and `screen` (iOS and server-side only) methods.
+
 You can integrate CleverTap using a server-side or mobile destination (iOS or Android). If you are interested in using CleverTap's push notifications or in-app notifications products, you should use the mobile destinations.
 
-All server-side destination requests require both the Segment `anonymousId` and `userId` in the payload. This is a requirement from CleverTap. CleverTap maintains the server-side integration. For any issues with the server-side integration, [contact the CleverTap Support team](https://help.clevertap.com/hc/en-us/requests/new){:target="_blank"}.
+All server-side destination requests require both the Segment `anonymousId` and `userId` in the payload. This is a requirement from CleverTap. 
 
-CleverTap supports the `identify`, `track`, `page` (server-side only), and `screen` (iOS and server-side only) methods.
+CleverTap maintains the server-side and mobile integrations:
+
```
Comment (niallzato) [inline src/connections/destinations/catalog/clevertap/index.md:17]: Can you delete this empty line? LGTM after that is done!

**Review by niallzato — CHANGES_REQUESTED** [review]

File: src/connections/destinations/catalog/clevertap/index.md [hunk 2]
```diff
@@ -7,12 +7,17 @@ cmode-override: true
 
 Once the Segment library is integrated, toggle CleverTap on in your Segment destinations, and add your CleverTap Account ID and CleverTap Account Token which you can find in the CleverTap Dashboard under Settings.
 
+CleverTap supports the `identify`, `track`, `page` (server-side only), and `screen` (iOS and server-side only) methods.
```
Comment (rchinn-segment) [inline src/connections/destinations/catalog/clevertap/index.md:10]:
```suggestion
CleverTap supports the Identify, Track, Page (server-side only), and Screen (iOS and server-side only) methods.
```

File: src/connections/destinations/catalog/clevertap/index.md [hunk 3]
```diff
@@ -7,12 +7,17 @@ cmode-override: true
...
+
 You can integrate CleverTap using a server-side or mobile destination (iOS or Android). If you are interested in using CleverTap's push notifications or in-app notifications products, you should use the mobile destinations.
 
-All server-side destination requests require both the Segment `anonymousId` and `userId` in the payload. This is a requirement from CleverTap. CleverTap maintains the server-side integration. For any issues with the server-side integration, [contact the CleverTap Support team](https://help.clevertap.com/hc/en-us/requests/new){:target="_blank"}.
+All server-side destination requests require both the Segment `anonymousId` and `userId` in the payload. This is a requirement from CleverTap.
```
Comment (rchinn-segment) [inline src/connections/destinations/catalog/clevertap/index.md:14]:
```suggestion
For server-side destination requests, CleverTap requires both the Segment `anonymousId` and `userId` in the payload.  
```

File: src/connections/destinations/catalog/clevertap/index.md [hunk 4]
```diff
@@ -7,12 +7,17 @@ cmode-override: true
...
+CleverTap supports the `identify`, `track`, `page` (server-side only), and `screen` (iOS and server-side only) methods.
+
 You can integrate CleverTap using a server-side or mobile destination (iOS or Android). If you are interested in using CleverTap's push notifications or in-app notifications products, you should use the mobile destinations.
 
-All server-side destination requests require both the Segment `anonymousId` and `userId` in the payload. This is a requirement from CleverTap. CleverTap maintains the server-side integration. For any issues with the server-side integration, [contact the CleverTap Support team](https://help.clevertap.com/hc/en-us/requests/new){:target="_blank"}.
+All server-side destination requests require both the Segment `anonymousId` and `userId` in the payload. This is a requirement from CleverTap. 
 
-CleverTap supports the `identify`, `track`, `page` (server-side only), and `screen` (iOS and server-side only) methods.
+CleverTap maintains the server-side and mobile integrations:
+- [Android](https://github.com/CleverTap/clevertap-segment-android)
```
Comment (rchinn-segment) [inline src/connections/destinations/catalog/clevertap/index.md:17]:
```suggestion
- [Android](https://github.com/CleverTap/clevertap-segment-android){:target="_blank"}
```

File: src/connections/destinations/catalog/clevertap/index.md [hunk 5]
```diff
@@ -7,12 +7,17 @@ cmode-override: true
...
+- [iOS](https://github.com/CleverTap/clevertap-segment-ios)
```
Comment (rchinn-segment) [inline src/connections/destinations/catalog/clevertap/index.md:18]:
```suggestion
- [iOS](https://github.com/CleverTap/clevertap-segment-ios){:target="_blank"}
```

**Review by rchinn-segment — CHANGES_REQUESTED** [review]
Looks good - just a few additional edits to align this with our [style guide](https://github.com/segmentio/segment-docs/blob/develop/styleguide.md#styling-the-segment-methods).

**Review by rchinn-segment — APPROVED** [review]

**Pull request merged**